caom2utils/caom2utils/caom2blueprint.py

```diff
@@ -5189,7 +5229,7 @@ def _get_common_arg_parser():
     fits2caom2 and caom2gen
     :return: args parser
     """
-    resource_id = "ivo://cadc.nrc.ca/fits2caom2"
+    resource_id = "ivo://cadc.nrc.ca/global/raven"
```

Comment:
Shouldn't be a module const?

Reply:
Defined and used as GLOBAL_STORAGE_RESOURCE_ID